src/Illuminate/Database/Eloquent/Attributes/UseFactory.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Illuminate\Database\Eloquent\Attributes;
+
+use Attribute;
+
+#[Attribute(Attribute::TARGET_CLASS)]
+class UseFactory
+{
+    /**
+     * Create a new attribute instance.
+     *
+     * @param  string  $factoryClass
```

Comment:
Shouldn't this be `class-string` instead of `string`?
```suggestion
     * @param  class-string  $factoryClass
```